storage/src/vespa/storage/distributor/pendingclusterstate.cpp

```diff
@@ -32,8 +32,8 @@ PendingClusterState::PendingClusterState(
         api::Timestamp creationTimestamp)
     : _cmd(newStateCmd),
       _requestedNodes(newStateCmd->getSystemState().getNodeCount(lib::NodeType::STORAGE)),
-      _prevClusterState(clusterInfo->getClusterState()),
-      _newClusterState(newStateCmd->getSystemState()),
+      _prevClusterStateBundle(clusterInfo->getClusterStateBundle()),
+      _newClusterStateBundle(newStateCmd->getSystemState()),
```

Comment:
Should this be `newStateCmd->getClusterStateBundle()` instead? Currently constructed from singular `ClusterState`